sale_elaboration/models/product_category.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    elaboration_profile_id = fields.Many2one(
+        comodel_name="product.elaboration.profile", string="Elaboration Profile"
+    )
+
+    def write(self, vals):
+        """Override update to set the elaboration profile on products."""
```

Comment:
Don't use docstrings on overriden methods as the docstring is overriden as well. Then when you inspect the method you'll get the last overriden string.

Use a regular comment instead and be concise avoiding the most obvious information.
```suggestion
        # Set the elaboration profile on the category products it apply just to the products with the exact category"""
```

Reply:
All atended.